### DISCUSSION TITLE
Do NOT use these discussions anymore!

### DISCUSSION THREAD

**Discussion by Navid200** [Navid200/xDrip#180] [Announcements]
Hello,

The discussions are now open in the main repository:
https://github.com/NightscoutFoundation/xDrip/discussions

You should use that platform instead of this one.
This platform will eventually be closed when all the outstanding discussions have either been addressed or transferred.
But, please don't open any new discussions here.

**Comment by emp-00** [comment]
Proposal: If the intention is now 100% clear closing "Navid200/xDrip/Discussions" and using "NightscoutFoundation/xDrip/Discussions" from now on:
1) Transfer all open+closed topics from Navid200 to Nightscoutfoundation
2) Close "Navid200/xDrip/Discussions" right away, thus avoiding opening of new topics etc.

Since it's both on github no user will lose data/access and the action should be doable within 1 day. 
Result: We all have a clean re-launch in the main repository without the confusion of two parallel forums...

**Reply by Navid200** [reply]
I don't have enough authorization for the main repository.  When I attempted to move one of the discussions, I was able to initiate.  Then, it asked me to identify the destination repository.  But, I could not choose the main repository.  I suspect because I don't have write permission into that repository.

I suspected that since @tolot27 has higher authorization than me, he may be able to do it.  So, I suggested it to him.
He stated that he could not do it either.

So, I went through all the discussions and identified the ones that the person who had opened it was happy with the outcome.  
I posted in the rest and suggested to the owner to copy it to the main repository.
If the same owner had multiple discussions, I made the suggestion only in one of their discussions.

**Reply by emp-00** [reply]
I'm surprised! Thought you guys had excellent connections so that this simple transfer of a bunch of discussions should be no "access problem". Have you requested transfer-write permissions and has this been denied?! Can't believe this. Looks like the main repository introduced the Discussions based on your good example and now they don't accept transferring the existing xDrip topics? Why not?

Actually I don't know how to manually "copy/paste" threads into the new forum without losing information/functionality/links etc...

**Reply by tolot27** [reply]
> I'm surprised! Thought you guys had excellent connections so that this simple transfer of a bunch of discussions should be no "access problem". 

I'm not really sure what you are trying to say with this statement. Nevertheless, the problem is discussed [elsewhere](https://github.com/github/feedback/discussions/3381) and I've contacted GitHub for clarification and support.

**Comment by Navid200** [comment]
When I suggested the discussions to be opened in the main repository, my reasons were:
1- The discussions can be converted to issues.
2- Issues can be converted to discussions.
3- Users may trust the discussions in the main repository while not want to use discussions in my repository.

Anything that happens in the main repository can be an additional responsibility/chore for jamorham.

He has enough on his plate:
- He has to choose which PR is merged and which one is not.  Every PR he does not merge can become a reason for someone to consider him to have a personal reason for making that choice.  That in itself is a lot of responsibility.
- He has written a major part of xDrip.  I keep going to him with questions when I need help.  That's a lot of work/time.
- Every time we need a change to how things should be done, we ask him to make a change.
- Anything that anyone is not happy about is sometimes brought up with him as a complaint.

That's too much considering he is also just a volunteer.  If he gets sick of this and stops contributing, we will be in serious trouble.  We really really need him and tzachi and tolot27 and the rest of the few developers who are still here.  We have to treat them with care and respect.  
When I suggested, again, to him to open the discussions in the main repository, one of my statements was: I will take care of anything that needs to be done; you don't have to spend any time on the administration.
I am going to live by that.  Even if it means I am going to be blamed for the time it may take us to copy the discussions.
So, I don't want anyone asking him to do it.

So:
- If you have a discussion here that you are satisfied with, please leave it be.  I am not going to delete any of the discussions.  I only intend to stop people from opening any new ones.
- If you have a discussions that has not concluded and still needs an answer, please open a new discussion in the main repository.  
If you feel there is content in the discussions here that you prefer not to copy but is important, include a link in the new discussion to the one here.
If you feel that you can include the gist of it in your new post in the new discussion, do that and let's forget about the one here.

If you have a different suggestion, I am all ears.


Edit: 
It turns out I will not be able to just leave these discussions open and not allow people to post new discussions in it.

**Reply by tolot27** [reply]
One more advantage of having discussions in the main repository is that they can be tagged with the same labels used for issues and PRs.

**Comment by tolot27** [comment]
One workaround to transfer the discussions is to (temporarily) transfer https://github.com/Navid200/xDrip to the organisation https://github.com/NightscoutFoundation. Maybe, this requires renaming the repository before. Afterwards, discussions can be transferred. If this has been done, the repository can be transferred back to Navid200.
I'll attach this possible workaround to the GitHub support ticket. Let us see what the support team answers.

**Reply by Navid200** [reply]
You mean anyone can move a personal repository into Nightscout?  How?

Can I create a new one there?
Shouldn't I then be able to transfer the discussions to that one?
Then, from there to the main repository?

**Reply by tolot27** [reply]
> You mean anyone can move a personal repository into Nightscout? How?

To NightscoutFoundation! I've corrected my previous post. Yes, everyone can do this but it requires interaction with the organisation owner ([bewest](https://github.com/bewest) in this case. He must accept the transfer. You need higher permissions afterwards for the transferred repository and https://github.com/NightscoutFoundation/xDrip to transfer the discussions. All a bit complicated.

John and I have contact to Ben West and he could assist us. But we should wait till GitHub Support responded because it is really hacky and may not work.

> Can I create a new one there?

No, you cannot until you are a member of that organisation.

> Shouldn't I then be able to transfer the discussions to that one? Then, from there to the main repository?

https://github.com/NightscoutFoundation is and organisation not a repository.

**Comment by Navid200** [comment]
| Discussion | Category | Category # | Should be copied |   
|-------------|------------|-------------|---------------------|  
| https://github.com/Navid200/xDrip/discussions/15 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/16 | " | 1 | |  
| https://github.com/Navid200/xDrip/discussions/18 | " | 1 | |  
| https://github.com/Navid200/xDrip/discussions/20 | Owner happy; diverged discussion | 1, 2 | |  
| https://github.com/Navid200/xDrip/discussions/22 | Owner has a suggestion | 13 | + |  
| https://github.com/Navid200/xDrip/discussions/24 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/26 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/28 | Unanswered question | 10 | + |  
| https://github.com/Navid200/xDrip/discussions/30 | Owner happy with conclusion  | 1 | |  
| https://github.com/Navid200/xDrip/discussions/44 | " | 1 | |  
| https://github.com/Navid200/xDrip/discussions/45 | " | 1 | |  
| https://github.com/Navid200/xDrip/discussions/47 | " | 1 | |  
| https://github.com/Navid200/xDrip/discussions/51 | No response from owner | 4 | |  
| https://github.com/Navid200/xDrip/discussions/53 | " | 4 | |  
| https://github.com/Navid200/xDrip/discussions/57 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/58 | Bug associated with an open issue |  5 | |  
| https://github.com/Navid200/xDrip/discussions/60 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/65 | " | 1 | |  
| https://github.com/Navid200/xDrip/discussions/66 | Sharing | 11 | + |  
| https://github.com/Navid200/xDrip/discussions/67 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/69 | " | 1 | |  
| https://github.com/Navid200/xDrip/discussions/71 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/72 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/74 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/75 | Issue opened | 5 | |  
| https://github.com/Navid200/xDrip/discussions/76 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/78 |  No response from owner | 4 | |  
| https://github.com/Navid200/xDrip/discussions/80 | Bug associated with an open issue | 5 | |  
| https://github.com/Navid200/xDrip/discussions/82 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/84 | Owner has a question | 12 | + |  
| https://github.com/Navid200/xDrip/discussions/86 | Bug associated with an open issue | 5 | |  
| https://github.com/Navid200/xDrip/discussions/87 | Bug associated with an open issue | 5 | |  
| https://github.com/Navid200/xDrip/discussions/89 | Bug associated with an open issue | 5 | |  
| https://github.com/Navid200/xDrip/discussions/91 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/93 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/94 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/95 | Feature request | 14 | + |  
| https://github.com/Navid200/xDrip/discussions/96 | Not concluded | 15 | + |  
| https://github.com/Navid200/xDrip/discussions/97 | Feature request | 14 | + |  
| https://github.com/Navid200/xDrip/discussions/98 | Feature request | 14 | + |  
| https://github.com/Navid200/xDrip/discussions/99 | Not concluded | 15 | + |  
| https://github.com/Navid200/xDrip/discussions/100 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/102 | Can be posted again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/103 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/105 | Just a question, which can be asked again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/107 | Not concluded | 15 | + |  
| https://github.com/Navid200/xDrip/discussions/109 | Just a question, which can be asked again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/110 | Just a question, which can be asked again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/111 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/113 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/115 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/116 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/118 | Just a question, which can be asked again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/120 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/121 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/124 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/125 | Can be posted again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/127 | No response from owner | 4 | |  
| https://github.com/Navid200/xDrip/discussions/129 | Not concluded | 15 | + |  
| https://github.com/Navid200/xDrip/discussions/130 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/131 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/133 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/135 | No response from owner | 4 | |  
| https://github.com/Navid200/xDrip/discussions/137 | Just a question, which can be asked again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/139 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/140 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/142 | Not concluded | 15 | + |  
| https://github.com/Navid200/xDrip/discussions/144 | No response from owner | 4 | |  
| https://github.com/Navid200/xDrip/discussions/146 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/148 | Bug associated with an open issue | 5 | |  
| https://github.com/Navid200/xDrip/discussions/150 | Issue opened | 5 | |  
| https://github.com/Navid200/xDrip/discussions/151 | Not concluded | 15 | + |  
| https://github.com/Navid200/xDrip/discussions/153 | Bug associated with an open issue | 5 | |  
| https://github.com/Navid200/xDrip/discussions/154 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/156 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/158 | No response from owner | 4 | |  
| https://github.com/Navid200/xDrip/discussions/159 | Feature request | 14 | + |  
| https://github.com/Navid200/xDrip/discussions/161 | Not concluded | 15 | + |  
| https://github.com/Navid200/xDrip/discussions/163 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/165 | Owner has a how-to | 16 | + |  
| https://github.com/Navid200/xDrip/discussions/167 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/169 | Can be posted again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/171 | Feature request | 14 | + |  
| https://github.com/Navid200/xDrip/discussions/172 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/174 | Seems to be concluded | 3 | |  
| https://github.com/Navid200/xDrip/discussions/175 | Owner happy with conclusion | 1 | |  
| https://github.com/Navid200/xDrip/discussions/177 | Can be posted again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/178 | Can be posted again | 6 | |  
| https://github.com/Navid200/xDrip/discussions/179 | Should be concluded here | 7 | |
